<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notes</class>
 <widget class="QMainWindow" name="notes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>623</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Corsiva Hebrew</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
    <property name="leftMargin">
     <number>11</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
      <property name="spacing">
       <number>7</number>
      </property>
      <item>
       <widget class="QLineEdit" name="noteName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textNote">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveChanges">
        <property name="inputMethodHints">
         <set>Qt::ImhHiddenText</set>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="addNew">
        <property name="text">
         <string>create context</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="archive">
        <property name="text">
         <string>show with archived</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeNotes">
        <property name="columnCount">
         <number>1</number>
        </property>
        <column>
         <property name="text">
          <string notr="true">✧YOUR NOTES✧</string>
         </property>
        </column>
        <item>
         <property name="text">
          <string>.｡･ﾟﾟ･all･ﾟﾟ･｡.</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>786</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>☆file☆</string>
    </property>
    <addaction name="actionLoad_all"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSort">
    <property name="title">
     <string>☆sort☆</string>
    </property>
    <addaction name="actionShow_by_time"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSort"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_note">
   <property name="text">
    <string>Add note</string>
   </property>
  </action>
  <action name="actionAdd_folder">
   <property name="text">
    <string>Add folder</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionShow_by_time">
   <property name="text">
    <string>･ﾟ✧show by time✧･ﾟ</string>
   </property>
  </action>
  <action name="actionstudy">
   <property name="text">
    <string>study</string>
   </property>
  </action>
  <action name="actionwork">
   <property name="text">
    <string>work</string>
   </property>
  </action>
  <action name="actionadd_new_type">
   <property name="text">
    <string>･ﾟ✧add new type✧･ﾟ</string>
   </property>
  </action>
  <action name="actionLoad_all">
   <property name="text">
    <string>･ﾟ✧load all✧･ﾟ</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
